trio/tests/test_sync.py

```diff
@@ -115,6 +115,21 @@
     c.release_on_behalf_of("value 1")
 
 
+async def test_CapacityLimiter_inf():
```

Comment:
The mixed CamelCase here probably breaks PEP-8: https://pep8.org/#function-names

Reply:
Yeah, PEP 8 is a set of guidelines to encourage consistency and readability, not strict rules that Guido wants you to follow mindlessly :-). (He actually got frustrated at the pep8 tool and asked its devs to rename it to "pycodestyle", because he didn't think PEP 8 *can* be enforced by an automatic tool.) Here, no one is going to get confused by the name into thinking it's anything besides a function, and the name is descriptive, so... works for me.